terraform/pagerduty/aws.tf

```diff
@@ -17,52 +17,54 @@ resource "aws_secretsmanager_secret" "pagerduty_integration_keys" {
+    ncas_non_prod_alarms            = pagerduty_service_integration.ncas_non_prod_cloudwatch.integration_key,
+    ncas_prod_alarms                = pagerduty_service_integration.ncas_prod_cloudwatch.integration_key,
+    wardship_non_prod_alarms        = pagerduty_service_integration.wardship_non_prod_cloudwatch.integration_key,
+    wardship_prod_alarms            = pagerduty_service_integration.wardship_prod_cloudwatch.integration_key,
+    pra_non_prod_alarms             = pagerduty_service_integration.pra_non_prod_cloudwatch.integration_key,
+    pra_prod_alarms                 = pagerduty_service_integration.pra_prod_cloudwatch.integration_key,
+    tipstaff_non_prod_alarms        = pagerduty_service_integration.tipstaff_non_prod_cloudwatch.integration_key,
+    tipstaff_prod_alarms            = pagerduty_service_integration.tipstaff_prod_cloudwatch.integration_key,
+    dacp_non_prod_alarms            = pagerduty_service_integration.dacp_non_prod_cloudwatch.integration_key,
+    dacp_prod_alarms                = pagerduty_service_integration.dacp_prod_cloudwatch.integration_key,
+    laa_maat_api_nonprod_alarms     = pagerduty_service_integration.laa_maat_api_nonprod_cloudwatch.integration_key,
+    laa_maat_api_prod_alarms        = pagerduty_service_integration.laa_maat_api_prod_cloudwatch.integration_key,
+    delius_core_nonprod_alarms      = pagerduty_service_integration.delius_core_nonprod_cloudwatch.integration_key
+    delius_oracle_nonprod_alarms    = pagerduty_service_integration.delius_oracle_nonprod_cloudwatch.integration_key
+    delius_nextcloud_nonprod_alarms = pagerduty_service_integration.delius_nextcloud_nonprod_cloudwatch.integration_key
+    delius_nextcloud_prod_alarms    = pagerduty_service_integration.delius_nextcloud_prod_cloudwatch.integration_key
+    laa_cwa_nonprod_alarms          = pagerduty_service_integration.cwa_non_prod.integration_key
+    laa_cwa_prod_alarms             = pagerduty_service_integration.cwa_prod.integration_key
+    laa_apex_nonprod_alarms         = pagerduty_service_integration.apex_non_prod.integration_key
+    laa_apex_prod_alarms            = pagerduty_service_integration.apex_prod.integration_key
+    # delius_mis_non_prod                     = pagerduty_event_orchestration_integration.delius_mis_non_prod_integration.parameters[0].routing_key
+    delius_mis_non_prod                     = pagerduty_service_integration.delius_mis_non_prod.integration_key
     delius_mis_prod_alarms                  = pagerduty_service_integration.delius_mis_prod.integration_key
     laa_edw_nonprod_alarms                  = pagerduty_service_integration.edw_non_prod.integration_key
     laa_edw_prod_alarms                     = pagerduty_service_integration.edw_prod.integration_key
+    cdpt_ifs_cloudwatch_routing_key         = pagerduty_event_orchestration_integration.cdpt_ifs_cloudwatch_integration.parameters[0].routing_key
```

Comment:
can we remove _routing_key from cdpt_ifs_cloudwatch_routing_key. to cdpt-ifs-alarms as that's how they call it in the environments file

Reply:
routing_key removed and code re-pushed

Reply:
Correct code re-pushed